### PR TITLE
53 3.13 load email drafts

### PR DESCRIPTION
**Pull request by timbasel**
Load Emails Templates

- Click on Template sets the hide_recipients radio buttons, subject and content field with template content

### CODE REVIEW

File: spec/models/ability_spec.rb [hunk 1]
```diff
@@ -302,4 +302,21 @@
     expect(ability).to be_able_to(:send_email, Email)
   end
 
+
+  %i[pupil coach].each do |role|
```
Comment (ekrebs5) [inline spec/models/ability_spec.rb]: These tests are already in the file, adding them would mean we have these tests 2-times

File: app/assets/javascripts/emails.js [hunk 2]
```diff
@@ -13,4 +13,21 @@ jQuery(function() {
             }
         }
     });
-});
\ No newline at end of file
+});
+
+function loadTemplate(elem) {
```
Comment (Lando-L) [inline app/assets/javascripts/emails.js]: Anstatt die Funktion ins Html reinzuschreiben würde ich mehr jQuery Funktionen nutzen und so vorgehen:
```
$('.email-template.list-group-item').click(function() {
  var subject = $(this).find('template-subject')[0].text()
  var content = ...
});
```

**Review by Lando-L — CHANGES_REQUESTED** [review]

**Review by Lando-L — APPROVED** [review]

File: spec/models/ability_spec.rb (hunk 1 above)
Comment (ekrebs5) [inline spec/models/ability_spec.rb]: Now as review: 
These tests are already in the file, adding them would mean we have these tests 2-times

**Review by ekrebs5 — DISMISSED** [review]

**Review by frederike-ramin — DISMISSED** [review]
You didn't add any test, right? One feature test could do the job.

Plus, I agree with @ekrebs5 about the doubled tests.

**Review by ekrebs5 — APPROVED** [review]

File: spec/factories/email_templates.rb [hunk 3]
```diff
@@ -14,17 +14,17 @@
...
-    status :acceptance
-  end
+    trait :default do
+      status :default
+    end
 
-  factory :email_template_rejection, parent: :email_template do
-    status :rejection
-  end
+    trait :accepted do
```
Comment (aloeser) [inline spec/factories/email_templates.rb]: you should call it acceptance

File: spec/factories/email_templates.rb [hunk 4]
```diff
@@ -14,17 +14,17 @@
...
+      status :acceptance
+    end
 
-  factory :email_template_default, parent: :email_template do
-    status :default
+    trait :rejected do
```
Comment (aloeser) [inline spec/factories/email_templates.rb]: you should call it rejection

**Review by aloeser — CHANGES_REQUESTED** [review]
just some small factory trait name changes, for consistency reasons

File: spec/features/email_spec.rb [hunk 5]
```diff
@@ -56,6 +56,18 @@
...
 
+  scenario "logged in as Organizer I can load an email template", js: true do
+    login(:organizer)
+    @template = FactoryGirl.create(:email_template, :acceptance)
+
+    visit event_email_show_path(@event, status: :acceptance)
+    first('.email-template').click
+
+    expect(page.find('#email_subject').value).to eq(@template.subject)
+    expect(page.find('#email_content').value).to eq(@template.content)
```
Comment (aloeser) [inline spec/features/email_spec.rb]: please also check for hide_recipients

**Review by aloeser — CHANGES_REQUESTED** [review]
Last small change, looks good otherwise :)

**Review by aloeser — APPROVED** [review]

**Pull request merged**